etc/acme/templates/cert.ctmpl

```diff
@@ -1 +1,2 @@
-{{if key "nginx/acme/cert"}}{{key "nginx/acme/cert"}}{{end}}
+{{ $service_name := env "SERVICE_NAME" }}
```

Comment:
Right, honestly I'm not sure how to do it and I didn't have time yesterday to look. Just needed it working to show off a few things